src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -60,13 +76,34 @@ public function iRenameItToIn(string $name, string $localeCode): void
                 $localeCode => [
                     'locale' => $localeCode,
                     'name' => $name,
+                    'slug' => StringInflector::nameToSlug($name)
```

Comment:
This step is related to the renaming of product. IMHO it is not part of this request to update slug

Reply:
Yes but in our UI slug is add automatically, and is necessarily in some tests:

<img width="1076" alt="Screenshot 2020-03-17 at 09 28 28" src="https://user-images.githubusercontent.com/39232096/76836987-07fe9600-6832-11ea-8194-9f7e72aeb091.png">

<img width="973" alt="Screenshot 2020-03-17 at 09 27 37" src="https://user-images.githubusercontent.com/39232096/76836984-0503a580-6832-11ea-95bf-a9310f43411c.png">

Reply:
That's my point. Right now, we will force UI to care about it, while IMHO we should provide the same behavior. Maybe a custom data provider will be a good fit here? Or some listener. /cc @GSadee @Zales0123

Reply:
Listener/persister that is adding the automatically created slug if a custom one is not passed 👌 Seems legit for me

Reply:
Ok but i will open new PR for it after merge